tools/experiments/experiments.js

```diff
@@ -235,12 +235,6 @@ const EXPERIMENTS = [
     name: 'Use slot width/height attribute for AdSense size format',
   },
   {
-    id: 'amp-form-verifiers',
```

Comment:
I'm just making sure, are we supposed to keep an experiment in this file and remove it in a later PR, or are these OK to remove in the same PR? I feel like I remember something about deleting experiments in two stages, but I could be mis-remembering